gitmessage

```diff
@@ -1,11 +1,20 @@
+# 50-character subject line
 
+# 72-character wrapped description.
 
-# 50-character subject line
-#
-# 72-character wrapped longer description. This should answer:
-#
-# * Why was this change necessary?
-# * How does it address the problem?
-# * Are there any side effects?
-#
-# Include a link to the ticket, if any.
+Why this is necessary:
+
+*
+*
+
+How it addresses the need:
+
+*
+*
+
+Side effects:
+
+*
+*
```

Comment:
Deleted the second bullet in each example so there's less to delete when editing the commit message.